rpms.in.yaml

```diff
@@ -0,0 +1,62 @@
+contentOrigin:
+  repos:
+    - repoid: rhel-9-for-$basearch-baseos-rpms
+      baseurl: https://cdn.redhat.com/content/dist/rhel9/{version}/$basearch/baseos/os
```

Comment:
how is {version} value is assigned? I understand this is a label in the dockerfile but what about the prefetch deps step how will it know what value should be set in $version?

Reply:
The prefetch deps task downloads the RPMs listed in the `rpms.lock.yaml` file. It is the `rpm-lockfile-prototype` that processes the `rpms.in.yaml` file.